pyteamcity/future/teamcity.py

```diff
@@ -79,3 +80,55 @@ def from_environ(cls):
             username=os.environ.get('TEAMCITY_USER'),
             password=os.environ.get('TEAMCITY_PASSWORD'),
             server=os.environ.get('TEAMCITY_HOST'))
+
+    @property
+    def server_info(self):
+        url = self.base_url + '/app/rest/server'
+        res = self.session.get(url)
+        res.raise_for_status()
```

Comment:
Wouldn't you want to wrap this into your own exception instead of exposing the exception of the underlying http library (requests)?